CONTRIBUTING.md

```diff
@@ -9,9 +9,28 @@ Thank you for your interest in the Animal Forest decompilation. This is a brief
 
 Most discussions happen in the `#af-decomp` channel on our [Discord Server](https://discord.zelda64.dev) where you are welcome to ask if you need help getting started.
 
-## Preparing a Pull Request
+## Tips for Successful Pull Requests
 
-You should work on a branch on your fork separate from your copy of main: it is always useful to have a clean main branch around if you need to fix something.
+- Don't bundle together multiple files worth of decompiled code into a single pr. Large prs require a lot of effort to review, and make things difficult for everyone.
+- You don't necessarily have to decompile a whole file, you can just decompile a couple of functions in a single pr if that suits your workflow better.
```

Comment:
The rest of the file says "PR" in uppercase, so I think it should be changed for consistency
```suggestion
- Don't bundle together multiple files worth of decompiled code into a single PR. Large PRs require a lot of effort to review, and make things difficult for everyone.
- You don't necessarily have to decompile a whole file, you can just decompile a couple of functions in a single PR if that suits your workflow better.
```